jerry-core/api/jerry.c

```diff
@@ -2050,6 +2050,34 @@ jerry_get_boolean_value (const jerry_value_t value) /**< api value */
   return ecma_is_value_true (value);
 } /* jerry_get_boolean_value */
 
+/**
+ * Check if the specified value is true.
+ *
+ * @return true or false.
```

Comment:
please keep the description in sync with the API docs (https://github.com/jerryscript-project/jerryscript/pull/4673/files#diff-6631cc2f42f3a11da30c3e679ed92cc619f1b3238066324b96a4d6a6f4264863R2168-R2169)